adg/pipelines/scripts/edx-run-test.sh

```diff
@@ -28,24 +28,76 @@ branch="$1"
+            -p no:randomly --cov --cov-config=.coveragerc-local \
+            --cov-report xml:reports/coverage-openedx.xml --cov-report html --disable-warnings
+            ;;
+        2)  # run tests for lms apps
+            python -Wd -m pytest --ds=lms.envs.test \
+            lms/djangoapps/__init__.py lms/djangoapps/badges lms/djangoapps/branding lms/djangoapps/bulk_email \
+            lms/djangoapps/bulk_enroll lms/djangoapps/ccx lms/djangoapps/certificates lms/djangoapps/commerce \
+            lms/djangoapps/course_api lms/djangoapps/course_blocks lms/djangoapps/course_goals \
+            lms/djangoapps/course_home_api lms/djangoapps/course_wiki lms/djangoapps/courseware \
+            lms/djangoapps/coursewarehistoryextended lms/djangoapps/dashboard lms/djangoapps/debug \
+            -p no:randomly --cov --cov-config=.coveragerc-local \
+            --cov-report xml:reports/coverage-lms1.xml --cov-report html --disable-warnings
+            ;;
+        3)  # run tests for lms apps
+            python -Wd -m pytest --ds=lms.envs.test \
+            lms/djangoapps/discussion lms/djangoapps/edxnotes \
+            lms/djangoapps/email_marketing lms/djangoapps/experiments lms/djangoapps/gating lms/djangoapps/grades \
+            lms/djangoapps/instructor \
+            -p no:randomly --cov --cov-config=.coveragerc-local \
+            --cov-report xml:reports/coverage-lms2.xml --cov-report html --disable-warnings
+            ;;
+        4)  # run tests for lms apps
+            python -Wd -m pytest --ds=lms.envs.test \
+            lms/djangoapps/instructor_analytics lms/djangoapps/instructor_task lms/djangoapps/learner_dashboard \
+            lms/djangoapps/lms_initialization lms/djangoapps/lms_xblock lms/djangoapps/lti_provider \
+            lms/djangoapps/mailing lms/djangoapps/mobile_api lms/djangoapps/monitoring lms/djangoapps/verify_student \
+            lms/djangoapps/program_enrollments lms/djangoapps/rss_proxy lms/djangoapps/shoppingcart \
+            lms/djangoapps/static_template_view lms/djangoapps/staticbook lms/djangoapps/support \
+            lms/djangoapps/survey lms/djangoapps/teams lms/djangoapps/tests lms/djangoapps/utils.py lms/lib \
+            -p no:randomly --cov --cov-config=.coveragerc-local \
+            --cov-report xml:reports/coverage-lms3.xml --cov-report html --disable-warnings
+            ;;
+        5)  # run all of the cms unit tests
+            paver test_system -s cms --cov-args="-p" --skip-clean --cov-config=.coveragerc-local --cov-report=xml:reports/coverage-cms.xml
+            ;;
+        6)  # run the commonlib unit tests
+            paver test_lib --cov-args="-p" --skip-clean --cov-config=.coveragerc-local --cov-report=xml:reports/coverage-lib.xml
+            ;;
+        *)
+            echo "No tests were executed in this container."
+            echo "Please adjust scripts/circle-ci-tests.sh to match your parallelism."
```

Comment:
Let's remove this as well or make the message more generic like "Please configure parallelism for better utilisation of containers"